gui/qt/qrreaderutil.py

```diff
@@ -0,0 +1,13 @@
+# TODO: Remove this when it is unlikely to encounter Qt5 installations that are
+# missing QtMultimedia
+def warn_unless_can_import_qrreader(parent):
```

Comment:
I don't want this file here -- if all it does is this. I'm merging this as-is and will push a commit on top.